Assignment-1/code/style.css

```diff
@@ -1 +1,57 @@
+.div1,.div2{
+    background-color:yellow ;
+    height:100px;
+    width: 45%;
+    display:inline-block;
+    margin:2%;
 
+}
+.div3{
+    background-color:pink ;
+    height:100px;
+    width: 95%;
+    display:block;
+    margin:2%;
+ 
+}
+.div4,.div5,.div6{
+    background-color:blue ;
+    height:100px;
+    width: 29%;
+    display:inline-block;
+    margin:2%;
+    margin-top: 2%;
+}
+/* media queries for 320px */
+@media screen and(max-width:320px){
+    .div1,.div2{
+        background-color:yellow ;
+        height:100px;
+        width: 45%;
+        display:inline-block;
+        margin:2%;
+    
+    }
+}
+@media screen and(max-width:768px){
+    .div1,.div2{
+        background-color:yellow ;
+        height:100px;
+        width: 45%;
+        display:inline-block;
+        margin:2%;
+    
+    }
+}
+@media screen and(max-width:768px){
```

Comment:
You are writing again @media query 2nd time for the 768px screen you can create once and use that only it will work.